### DISCUSSION TITLE
挂载pikpak网盘 怎么使用加速地址呢

### DISCUSSION THREAD

**Discussion by hello13485** [AlistGo/alist#7646] [Q&A]
挂载pikpak网盘 怎么使用加速地址呢  文档里说的 开启使用加速地址选项 怎么开启呢

**Comment by anwen-anyi** [comment]
已经作废了 文档还没来得及修改
- https://github.com/AlistGo/alist/pull/7530

**Reply by hello13485** [reply]
好吧 那是不是就无法在加速了呢
